modules/apps/item-selector/item-selector-taglib/src/main/java/com/liferay/item/selector/taglib/servlet/taglib/GroupSelectorTag.java

```diff
@@ -155,13 +155,15 @@ private List<Group> _getGroups(HttpServletRequest httpServletRequest) {
 	}
 
 	private int _getGroupsCount(HttpServletRequest httpServletRequest) {
+		String groupType = _getGroupType(httpServletRequest);
+
```

Comment:
Can you inline this call? `_getGroupType` caches its value, so it should make no difference.

In the conditional, you may use `Objects.equals` to avoid creating intermediary variables.